api/exec/helper.go

```diff
@@ -0,0 +1,15 @@
+package exec
+
+import (
+	"path"
+
+	portainer "github.com/portainer/portainer/api"
+)
+
+func getStackFilePaths(stack *portainer.Stack) []string {
```

Comment:
maybe this should be inside `internal/stacks`? like we have helpers for endpoints?